src/main/java/de/slimecloud/slimeball/features/alerts/youtube/GuildYoutubeConfig.java

```diff
@@ -3,39 +3,46 @@
 import de.slimecloud.slimeball.config.ConfigCategory;
 import de.slimecloud.slimeball.config.engine.ConfigField;
 import de.slimecloud.slimeball.config.engine.ConfigFieldType;
+import de.slimecloud.slimeball.config.engine.Info;
 import lombok.Getter;
 import net.dv8tion.jda.api.entities.Role;
 import net.dv8tion.jda.api.entities.channel.middleman.MessageChannel;
 import org.jetbrains.annotations.NotNull;
 
+import java.util.HashMap;
+import java.util.Map;
 import java.util.Optional;
 
 @Getter
 public class GuildYoutubeConfig extends ConfigCategory {
 	@ConfigField(name = "Kanal", command = "channel", description = "Der Text Kanal in den neue Videos gesendet werden", type = ConfigFieldType.MESSAGE_CHANNEL)
-	private Long channel;
+	@Info(keyType = ConfigFieldType.STRING)
+	private Map<String, Long> channel = new HashMap<>();
 
 	@ConfigField(name = "Rolle", command = "role", description = "Die Rolle, die bei neuen Videos gepingt wird", type = ConfigFieldType.ROLE)
-	private Long role;
+	@Info(keyType = ConfigFieldType.STRING)
+	private Map<String, Long> role = new HashMap<>();
 
 	@ConfigField(name = "Live Nachricht", command = "live-msg", description = "Die Nachricht, die bei livestreams gesendet wird; Placeholder: %role% %uploader% %url% %title%", type = ConfigFieldType.STRING)
-	private String liveMessage;
+	@Info(keyType = ConfigFieldType.STRING)
+	private Map<String, String> liveMessage = new HashMap<>();
 
 	@ConfigField(name = "Video Nachricht", command = "video-msg", description = "Die Nachricht, die bei neuen Videos gesendet wird; Placeholder: %role% %uploader% %url% %title%", type = ConfigFieldType.STRING)
-	private String videoMessage;
+	@Info(keyType = ConfigFieldType.STRING)
+	private Map<String, String> videoMessage = new HashMap<>();
 
 	@NotNull
-	public Long getChannelId() {
-		return channel;
+	public Optional<Long> getChannelId(@NotNull String youtubeChannelId) {
+		return Optional.ofNullable(channel.get(youtubeChannelId));
 	}
 
 	@NotNull
-	public Optional<MessageChannel> getChannel() {
-		return Optional.ofNullable(bot.getJda().getChannelById(MessageChannel.class, channel));
+	public Optional<MessageChannel> getChannel(@NotNull String youtubeChannelId) {
+		return Optional.ofNullable(bot.getJda().getChannelById(MessageChannel.class, channel.get(youtubeChannelId)));
```

Comment:
`This should be getChannelId(youtubeChannelId).flatMap(id -> bot.getJda().getChannelById(MessageChannel.class, id));` because this could otherwise throw an exception when you pass `null` to getChannelById